### PR TITLE
reconcile translations after extracting escape characters fix

### PR DESCRIPTION
**Pull request by sharifelgamal**
This is fixing the translation files after merging #13739

### CODE REVIEW

File: translations/de.json [hunk 1]
```diff
@@ -640,12 +637,11 @@
 	"The \"{{.driver_name}}\" driver should not be used with root privileges. If you wish to continue as root, use --force.": "",
 	"The \"{{.name}}\" cluster has been deleted.": "Der Cluster \"{{.name}}\" wurde gelöscht.",
 	"The \"{{.name}}\" cluster has been deleted.__1": "Der Cluster \"{{.name}}\" wurde gelöscht.",
-	"The \"{{.name}}\" container runtime requires CNI": "",
+	"The \"{{.name}}\" container runtime requires CNI": "Die Container Runtime \\\"{{.name}}\\\" erfordert ein CNI",
```
Comment (spowelljr) [inline translations/de.json:640]:
```suggestion
	"The \"{{.name}}\" container runtime requires CNI": "Die Container Runtime \"{{.name}}\" erfordert ein CNI",
```

File: translations/de.json [hunk 2]
```diff
@@ -996,8 +984,7 @@
 	"stat failed": "state Fehler",
 	"status json failure": "Status json Fehler",
 	"status text failure": "Status text Fehler",
-	"toom any arguments ({{.ArgCount}}).\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "",
-	"toom any arguments ({{.ArgCount}}).\\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "Zu viele Parameter ({{.ArgCount}}).\\nVerwendung: minikube config set PROPERTY_NAME PROPERTY_VALUE",
+	"too many arguments ({{.ArgCount}}).\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "Zu viele Parameter ({{.ArgCount}}).\\nVerwendung: minikube config set PROPERTY_NAME PROPERTY_VALUE",
```
Comment (spowelljr) [inline translations/de.json:987]:
```suggestion
	"too many arguments ({{.ArgCount}}).\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "Zu viele Parameter ({{.ArgCount}}).\nVerwendung: minikube config set PROPERTY_NAME PROPERTY_VALUE",
```

File: translations/fr.json [hunk 3]
```diff
@@ -965,8 +950,7 @@
 	"stat failed": "stat en échec",
 	"status json failure": "état du JSON en échec",
 	"status text failure": "état du texte en échec",
-	"toom any arguments ({{.ArgCount}}).\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "",
-	"toom any arguments ({{.ArgCount}}).\\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "toom tous les arguments ({{.ArgCount}}).\\nusage : jeu de configuration de minikube PROPERTY_NAME PROPERTY_VALUE",
+	"too many arguments ({{.ArgCount}}).\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "trop d'arguments ({{.ArgCount}}).\\nusage : jeu de configuration de minikube PROPERTY_NAME PROPERTY_VALUE",
```
Comment (spowelljr) [inline translations/fr.json:953]:
```suggestion
	"too many arguments ({{.ArgCount}}).\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "trop d'arguments ({{.ArgCount}}).\nusage : jeu de configuration de minikube PROPERTY_NAME PROPERTY_VALUE",
```

File: translations/ja.json [hunk 4]
```diff
@@ -993,8 +983,7 @@
 	"namespaces to unpause": "停止を解除する名前空間",
 	"network to run minikube with. Now it is used by docker/podman and KVM drivers. If left empty, minikube will create a new network.": "minikube を実行するネットワーク。現時点では docker/podman と KVM ドライバーで使用されます。空の場合、minikube は新しいネットワークを作成します。",
 	"none driver does not support multi-node clusters": "none ドライバーはマルチノードクラスターをサポートしていません",
-	"not enough arguments ({{.ArgCount}}).\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "",
-	"not enough arguments ({{.ArgCount}}).\\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "引数 ({{.ArgCount}}) が不十分です。\\n使用方法: minikube config set PROPERTY_NAME PROPERTY_VALUE",
+	"not enough arguments ({{.ArgCount}}).\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "引数 ({{.ArgCount}}) が不十分です。\\n使用方法: minikube config set PROPERTY_NAME PROPERTY_VALUE",
```
Comment (spowelljr) [inline translations/ja.json:986]:
```suggestion
	"not enough arguments ({{.ArgCount}}).\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "引数 ({{.ArgCount}}) が不十分です。\n使用方法: minikube config set PROPERTY_NAME PROPERTY_VALUE",
```

File: translations/ja.json [hunk 5]
```diff
@@ -1013,8 +1002,7 @@
 	"stat failed": "stat に失敗しました",
 	"status json failure": "status json に失敗しました",
 	"status text failure": "status text に失敗しました",
-	"toom any arguments ({{.ArgCount}}).\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "",
-	"toom any arguments ({{.ArgCount}}).\\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "引数 ({{.ArgCount}} 個) が多すぎます。\\n使用法: minikube config set PROPERTY_NAME PROPERTY_VALUE",
+	"too many arguments ({{.ArgCount}}).\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "引数 ({{.ArgCount}} 個) が多すぎます。\\n使用法: minikube config set PROPERTY_NAME PROPERTY_VALUE",
```
Comment (spowelljr) [inline translations/ja.json:1005]:
```suggestion
	"too many arguments ({{.ArgCount}}).\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "引数 ({{.ArgCount}} 個) が多すぎます。\n使用法: minikube config set PROPERTY_NAME PROPERTY_VALUE",
```

**Review by spowelljr — CHANGES_REQUESTED** [review]

File: translations/fr.json [hunk 6]
```diff
@@ -721,8 +715,7 @@
...
 	"To connect to this cluster, use: kubectl --context={{.profile_name}}": "Pour vous connecter à ce cluster, utilisez : kubectl --context={{.profile_name}}",
 	"To disable beta notices, run: 'minikube config set WantBetaUpdateNotification false'": "Pour désactiver les notifications bêta, exécutez : 'minikube config set WantBetaUpdateNotification false'",
-	"To disable this notice, run: 'minikube config set WantUpdateNotification false'\n": "",
-	"To disable this notice, run: 'minikube config set WantUpdateNotification false'\\n": "Pour désactiver cette notification, exécutez : 'minikube config set WantUpdateNotification false'\\n",
+	"To disable this notice, run: 'minikube config set WantUpdateNotification false'\n": "Pour désactiver cette notification, exécutez : 'minikube config set WantUpdateNotification false'\n",
 	"To disable update notices in general, run: 'minikube config set WantUpdateNotification false'\n": "",
 	"To disable update notices in general, run: 'minikube config set WantUpdateNotification false'\\n": "Pour désactiver les notifications de mise à jour en général, exécutez : 'minikube config set WantUpdateNotification false'\\n",
```
Comment (spowelljr) [inline translations/fr.json:720]:
```suggestion
```

**Review by spowelljr — CHANGES_REQUESTED** [review]

File: translations/fr.json (hunk 6 above)
Comment (sharifelgamal) [reply translations/fr.json]: I fixed this in a different way.

**Pull request merged**